setup.py

```diff
@@ -43,10 +43,10 @@
 
 PACKAGE = {
     'name': 'vsc-base',
-    'version': '2.4.9',
+    'version': '2.4.9.1',
     'author': [sdw, jt, ag, kh],
     'maintainer': [sdw, jt, ag, kh],
-    'packages': ['vsc.utils'],
+    'packages': ['vsc', 'vsc.utils'],
```

Comment:
OK, but `vsc` should be listed here (and in all other `vsc` packages).

`vsc-mympirun` has it too, for example: https://github.com/hpcugent/vsc-mympirun/blob/master/setup.py#L43

So does `vsc-utils`: https://github.com/hpcugent/vsc-utils/blob/master/setup.py#L57